### DISCUSSION TITLE
Search box in CRUD

### DISCUSSION THREAD

**Discussion by blondie63** [Laravel-Backpack/community-forum#888] [Q&A (Help)] [answered]
By deafult Search box is active for all CRUDs but i don't understand why in some cases not:
<img width="1473" alt="BKP-NO-search" src="https://github.com/Laravel-Backpack/community-forum/assets/1261851/1df5348e-55b6-4b61-9ab1-b913f6efcb70">

I'm working on some code that an ex of mine wrote. collaborator and I don't see any command that can disable the search, but it doesn't appear.. What could it be?

**Comment by o15a3d4l11s2** [comment]
The search field is part of [DataTables](https://datatables.net/manual/options#Setting-defaults) and not directly configurable as far as I remember. Inspect the page and try to find if there is some CSS/JS that hides it?

**Comment by jcastroa87** [comment, marked as answer]
Hello @blondie63 

We don't have a command to hide the search box, so there could be two ways:

1. Some JS to hide it
2. You are using a custom /resources/views/crud/list.blade.php and hide or remove the search box

Can you check if one of these cases happened to you?

Cheers.

**Reply by blondie63** [reply]
Yesss ! You are right !
He set $this->crud->setListView('crud::list.customers');

Thanks !!

**Comment by jcastroa87** [comment]
I'm glad you solve it, I will close the issue, but please feel free to re-open or create a new one.

Cheers.
